server/mail_templates/ingestDatasetConfirmation.mako

```diff
@@ -0,0 +1,18 @@
+<%include file="_header.mako"/>
+
+% if group:
+<%include file="ingestDatasetConfirmationHeaderGroup.mako"/>
```

Comment:
Does it make sense to template these headers separately? I don't think they'll ever be reusable components.